SPECS/lua/lua.spec

```diff
@@ -216,6 +212,9 @@ popd
 %{_libdir}/*.a
 
 %changelog
+* Tue Feb 27 2024 Andrew Phelps <anphel@microsoft.com> - 5.4.6-1
+- Upgrade to version 5.4.6
```

Comment:
Could you also mentioned we disabled bootstrap mode? It's going to be easier to find in the history, if we later learn we need to re-enable it.

Reply:
sure, will do